adapters/clickhouse.go

```diff
@@ -190,11 +190,17 @@ type ClickHouse struct {
 	dataSource            *sql.DB
 	tableStatementFactory *TableStatementFactory
 	nullableFields        map[string]bool
+	queryLogger           *logging.QueryLogger
 }
 
 //NewClickHouse return configured ClickHouse adapter instance
 func NewClickHouse(ctx context.Context, connectionString, database, cluster string, tlsConfig map[string]string,
-	tableStatementFactory *TableStatementFactory, nullableFields map[string]bool) (*ClickHouse, error) {
+	tableStatementFactory *TableStatementFactory, nullableFields map[string]bool,
+	queryLogger *logging.QueryLogger) (*ClickHouse, error) {
+	if queryLogger == nil {
```

Comment:
It would be better to create test instances with an empty logger and assume that queryLogger parameter can't be nil

Reply:
fxied